farmdata2_modules/fd2_tabs/resources/dropdownWithAllComponent.js

```diff
@@ -0,0 +1,54 @@
+/**
+ * Vue.js component that defines the dropdown menu.
+ */ 
+let DropdownWithAllComponent = {
+    template: `<div data-cy="dropdown-component">
+            <label for="dropdownOptions"><slot>Select: </slot></label>
+            <input list="dropdownOptions" v-model="selectedOption" data-cy="dropdown-input" @change="selectionChanged">
+                <datalist id="dropdownOptions" data-cy="dropdownOptions">
+                   <option v-for="singleOption in fullDropdown" data-cy="singleOption">{{ singleOption }}</option>
+                </datalist>
+        </div>`, 
+    props: {
+        dropdownList: {
+            type: Array,
+            required: true,
+            default: []
```

Comment:
Let's check on how required and default interact.  Seems if the prop is required to be provided by the html then the default would not be relevant.  But I'm not 100% on that.  Do a little research and resolve as appropriate.

Reply:
The default is not necessary when the prop is required, this was just a remnant from some debugging. I'll remove it.